extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -332,7 +335,7 @@ angular.module('oppia').component('svgFilenameEditor', {
           'data:image/svg+xml;base64,' +
           btoa(unescape(encodeURIComponent(svgString))));
         var invalidTagsAndAttr = (
-          ImageUploadHelperService.getInvalidSvgTagsAndAttrs(dataURI));
+          SvgSanitizerService.getInvalidSvgTagsAndAttrs(dataURI));
```

Comment:
Suggest changing the API to SvgSanitizerService.validate(dataURI) to centralize the validation. It should be possible for other services to treat the validation as a black box.

Reply:
Done!